subsys/settings/src/settings_init.c

```diff
@@ -59,7 +71,7 @@ static void settings_init_fcb(void)
 	int rc;
 	const struct flash_area *fap;
 
-	rc = flash_area_get_sectors(DT_FLASH_AREA_STORAGE_ID, &cnt,
+	rc = flash_area_get_sectors(FLASH_AREA_STORAGE_ID, &cnt,
```

Comment:
Also needed in `zephyr/subsys/settings/src/settings_fcb.c` at line 45.

Reply:
Fixed.